Tour/basic-types.md

```diff
@@ -169,9 +169,9 @@ _Note: indexing a buffer outside of its bounds is either caught as a compile-tim
 The type of a buffer is written either `T[n]` or `Buffer<T, n>`, where `T` is a type and `n` the number of elements in the buffer. All elements of a buffer must be initialized at the same time as the buffer itself, either by the means of a buffer literal expression, as in the program above, or by calling a buffer _initializer_:
 
 ```
-typealias Point = (x: Double, y: Double)
+typealias Point = (x: Float64, y: Float64)
```

Comment:
There are a lot of errors like this in the remaining examples. I'm assuming it's just not implemented yet.
```
.build/release/hc Examples/test.hylo -o test && ./test
Examples/test.hylo:1.19-43: error: expected type but expression denotes a value
typealias Point = (x: Float64, y: Float64)
                  ~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
This one is because of changes we made to the syntax. Tuple _types_ are written with `{...}`, not `(...)`.

Reply:
The alias should be written `{x: Float64, y: Float64}` and then you'll likely encounter another error, because accessing the field of a tuple by its name hasn't been implemented.

Reply:
> The alias should be written `{x: Float64, y: Float64}` and then you'll likely encounter another error, because accessing the field of a tuple by its name hasn't been implemented.

Yes, that's exactly what happened. 👍 
```
error: cannot call value of type 'Metatype<Point[3]>' as a function
  var triangle = Point[3](fun(i) { (x: Float64(i), y: 0.0) }) // <== HERE
```

Reply:
Updated tuple type syntax: https://github.com/hylo-lang/GitBooks/pull/13/commits/e383ef9ff9dd338a1f463225f9f65c3f7bff8318

Reply:
The spec looks like it's already correct regarding tuple type expressions: https://github.com/hylo-lang/specification/blob/main/spec.md#tuple-type-expressions